packages/driver/src/cy/retries.ts

```diff
@@ -116,7 +115,7 @@ export const create = (Cypress: ICypress, state: StateFunc, timeout: $Cy['timeou
       // bug in bluebird with not propagating cancellations
       // fast enough in a series of promises
       // https://github.com/petkaantonov/bluebird/issues/1424
-      return state('canceled') || state('error') || runnableHasChanged()
+      return state('canceled') || runnableHasChanged()
```

Comment:
Any idea if this code have any impact on before/beforeEach blocks? We have quite a few tests for this I believe in https://github.com/cypress-io/cypress/blob/after-hook-action-fix/packages/app/cypress/e2e/runner/runner.mochaEvents.cy.ts so it appears to not. Usually if the `before`/`beforeEach` fails I believe we skip the suite but I don't think that marks the runnable as canceled, but has the same effect?

I don't think we are going to know the true impact of this until we release it to the general public which might introduce the use/test case we are missing here... or it may not introduce anything at all except a fix!

Reply:
It shouldn't have any negative side effects because the code running in hook runnables is (and should be running) - its just that commands weren't retrying.